docroot/profiles/favrskov/themes/favrskovtheme/template/Views/job/views-view-table--job--job-listing-by-table.tpl.php

```diff
@@ -41,23 +44,21 @@
       <?php $destination_link = $row['field_job_destination_link']; ?>
       <?php unset($row['field_job_destination_link']); ?>
     <?php endif; ?>
-
-    <tr <?php if ($row_classes[$row_count]) { print 'class="' . implode(' ', $row_classes[$row_count]) .'"';  } ?>>
+    <tr style="cursor: pointer;" class='clickable-row' data-href='<?php print $destination_link; ?>' <?php if ($row_classes[$row_count]) { print 'class="' . implode(' ', $row_classes[$row_count]) .'"';  } ?>>
```

Comment:
Again css should be moved out to proper file, use id or class.

Reply:
Style moved to scss file.